integration-tests/src/test/scala/tech/beshu/ror/integration/suites/audit/RemoteClusterAuditingToolsSuite.scala

```diff
@@ -61,4 +62,10 @@ class RemoteClusterAuditingToolsSuite
   override protected def baseRorConfig: String = resolvedRorConfigFile.contentAsString
 
   override protected def baseAuditDataStreamName: Option[String] = Option.when(isDataStreamSupported)("audit_data_stream")
+
+  // Adding the ES cluster fields is enabled in the /cluster_auditing_tools/readonlyrest.yml config file (`DefaultAuditLogSerializerV2` is used)
+  override def assertForEveryAuditEntry(entry: JSON): Unit = {
```

Comment:
I'm not sure if this test covers this use case too, but I'd be nice to test the scenario as above:
```
1. The user starts ES with ROR with `DefaultAuditLogSerializerV1` configured
2. The user generates some traffic to create some audit documents
3. The user reconfigures ROR to use `DefaultAuditLogSerializerV2` and reloads config
4. The user generates some traffic to create some audit documents
```

The result - there should be no problem with the scenario above.